build.proj

```diff
@@ -67,6 +67,12 @@
     <ValidateExactRestore ProjectLockJsons="@(ProjectJsonsExcludingTemplateFiles->'%(RootDir)%(Directory)%(Filename).lock.json')" />
   </Target>
 
+  <Target Name="InitializeShippedNuGetPackage" BeforeTargets="UpdatePublishedVersions" >
+    <ItemGroup>
+      <ShippedNuGetPackage Include="$(PackagesOutDir)/**/*.nupkg" />
```

Comment:
To handle this corefx just passed the ShippedNuGetPackageGlobPath property to https://github.com/dotnet/corefx/blob/b5ac646f30e2d36729bdb95dac072ceb1e5b186d/buildpipeline/DotNet-Trusted-Publish.json#L177